pom.xml

```diff
@@ -74,12 +77,18 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>aws-credentials</artifactId>
-      <version>191.vcb_f183ce58b_9</version>
+      <exclusions>
+        <!-- TODO JCLOUDS-1620 -->
```

Comment:
https://issues.apache.org/jira/browse/JCLOUDS-1620 FTR. Serious problems would I hope get caught by https://github.com/jenkinsci/artifact-manager-s3-plugin/blob/87eb5414d596fc3965339afaa587cc5cff633465/src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/MinioIntegrationTest.java#L73 Supposedly fixed in 2.6.0 https://github.com/jenkinsci/artifact-manager-s3-plugin/blob/29d44d48bfa7a9bc5a0b66c845be4739edfaaaef/pom.xml#L66-L68

Reply:
How does this relate to #447?